### PR TITLE
altair: Add `orml_asset_registry`

### PR DESCRIPTION
**Pull request by NunoAlexandre**
##### altair
- [x] Repeat all adequate steps performed for `development` @ #829 
- [x] Extend the XCM tests to cover registered foreign assets

##### general 
 - [x] Drop `KUSD` in favour of `AUSD` #836 
     See final reason for decision: https://github.com/centrifuge/centrifuge-chain/issues/836#issuecomment-1161527544
     
 Closes https://github.com/centrifuge/centrifuge-chain/issues/836

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 on:
   push:
-    branches: [master, parachain]
+    branches: [master, parachain, altair/asset-registry]
```
Comment (mikiquantum) [inline .github/workflows/build.yml:3]: I guess we can drop this right?

**Review by mikiquantum — APPROVED** [review]
LGTM!

**Review by mikiquantum — APPROVED** [review]
We need to change the altair/Cargo.toml version and the node version too 👍

File: .github/workflows/build.yml (hunk 1 above)
Comment (NunoAlexandre) [reply .github/workflows/build.yml]: yes sir! 👍

**Comment by NunoAlexandre** [comment]
@mikiquantum thanks a lot for the review! Dropped the temporary CI changes and bumped the altair and node versions 👍

**Pull request merged**